packages/test/src/integration-tests.ts

```diff
@@ -1207,6 +1211,68 @@ export function runIntegrationTests(codec?: PayloadCodec): void {
     at stackTracer (test/src/workflows/stack-tracer.ts)`
       );
     });
+
+    test('Enhanced stack trace returns trace that makes sense', async (t) => {
+      const { client } = t.context;
+      const workflowId = uuid4();
+
+      const enhancedStack = await client.execute(workflows.enhancedStackTracer, {
+        taskQueue: 'test',
+        workflowId,
+      });
+
+      const stacks = enhancedStack.stacks.map((s) => ({
+        locations: s.locations.map((l) => ({
+          ...l,
+          ...(l.filePath ? { filePath: l.filePath.replace(path.resolve(__dirname, '../../../'), '') } : undefined),
```

Comment:
I'd make sure that the normalization done here is identical to what we have in `helper.ts#cleanStackTrace()`.